Code/Framework/AzCore/AzCore/std/containers/vector.h

```diff
@@ -5,13 +5,15 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
  */
+#pragma once
 #ifndef AZSTD_VECTOR_H
 #define AZSTD_VECTOR_H 1
```

Comment:
The #ifdef for the file guard can removed as well.

Reply:
Didn't want to step on any toes, since code guide states `Do not however, remove them from existing files unless you are already making changes near the header.` and I wasn't sure what counts as changes 'near the header' 😄

Reply:
Well `#pragma once` is near the header, so I think you should go ahead and remove the `#ifndef... #define ... #endif` guards as well

Reply:
Cleaned this one up.